frontend/projects/upgrade/src/app/core/experiments/store/experiments.reducer.ts

```diff
@@ -90,7 +94,11 @@ const reducer = createReducer(
     isLoadingExperiment: true,
   })),
   on(experimentsAction.actionGetExperimentByIdSuccess, (state, { experiment }) =>
-    adapter.upsertOne(experiment, { ...state, isLoadingExperiment: false })
+    adapter.upsertOne(experiment, {
+      ...state,
+      isLoadingExperiment: false,
+      totalExperiments: state.totalExperiments + 1,
```

Comment:
This seems to result in the count being one more than what the experiment total is. If you're not adding an experiment, just navigating to the experiment detail page, `actionGetExperimentByIdSuccess` is triggered so that the experiment whose name you just clicked on (which would have already been counted as part of the total) gets added again to the total count. Maybe this is currently not an issue because the only thing `state.totalExperiments` is used for is to determine whether to show the `noExperimentTemplate`. But I think if we ever want to depend on the accuracy of `state.totalExperiments` it might be an issue.